frontend/controller/actions/group.js

```diff
@@ -348,7 +348,7 @@ export default (sbp('sbp/selectors/register', {
       const state = rootState[params.contractID]
 
       // If we are expecting to receive keys, set up an event listener
-      if (sendKeyRequest || !state._volatile?.pendingKeyRequests?.length) {
+      if (sendKeyRequest || state._volatile?.pendingKeyRequests?.length) {
```

Comment:
Can you undo this change and apply commit a57c68f1424248326c4797949240a827d50c1858 instead?